browser-extension/mv2/src/client/js/sessionRecorder.js

```diff
@@ -114,11 +125,12 @@ RQ.SessionRecorder.addMessageListeners = () => {
     } else if (event.data.action === "sessionRecordingStarted") {
       RQ.SessionRecorder.isRecording = true;
 
-      if (RQ.SessionRecorder.showWidget) {
-        chrome.runtime.sendMessage({
-          action: RQ.CLIENT_MESSAGES.NOTIFY_SESSION_RECORDING_STARTED,
-        });
+      chrome.runtime.sendMessage({
+        action: RQ.CLIENT_MESSAGES.NOTIFY_SESSION_RECORDING_STARTED,
+        markRecordingIcon: RQ.SessionRecorder.showWidget || RQ.SessionRecorder.markRecordingIcon,
```

Comment:
The function of `showWidget` is different, shouldn't mix it with marking recording icon
```suggestion
        markRecordingIcon: RQ.SessionRecorder.markRecordingIcon,
```
Then `handleSessionRecordingOnClientPageLoad` in background should update as : 
```
if (sessionRecordingConfig?.autoRecording?.mode === "allPages") {
        sessionRecordingData.showWidget = false;
        sessionRecordingData.markRecordingIcon = false;
      }else{
        sessionRecordingData.showWidget = true;
      }
```